patches/minecraft/net/minecraft/data/registries/RegistriesDatapackGenerator.java.patch

```diff
@@ -1,6 +1,32 @@
 --- a/net/minecraft/data/registries/RegistriesDatapackGenerator.java
 +++ b/net/minecraft/data/registries/RegistriesDatapackGenerator.java
-@@ -31,7 +_,7 @@
+@@ -18,20 +_,33 @@
+ import net.minecraft.resources.ResourceKey;
+ import org.slf4j.Logger;
+ 
++/**
++ * @deprecated Forge: Use {@link net.minecraftforge.common.data.DatapackBuiltinEntriesProvider} instead
++ */
++@Deprecated
+ public class RegistriesDatapackGenerator implements DataProvider {
+    private static final Logger f_254654_ = LogUtils.getLogger();
+    private final PackOutput f_254743_;
+    private final CompletableFuture<HolderLookup.Provider> f_254747_;
++   private final java.util.function.Predicate<String> namespacePredicate;
+ 
++   @Deprecated
+    public RegistriesDatapackGenerator(PackOutput p_256643_, CompletableFuture<HolderLookup.Provider> p_255780_) {
+       this.f_254747_ = p_255780_;
```

Comment:
This constructor should instead delegate to the other one.

Reply:
The constructor takes in set of mod ids while the predicate always returns true. Are you suggesting I pass in all the mod ids or return null to force the set in that case?